feat(dedibox): smart tests (#4360)
scaleway/docs-content: dedibox/hardware/troubleshooting/diagnose-defective-disk.mdx

- A Dedibox account logged into the [console](https://console.online.net)
- A [Dedibox dedicated server](https://www.scaleway.com/en/dedibox/)

## How to check server with no RAID controller

1. Log into your server using SSH.
2. Run the following command from the root account (or precede it with `sudo`):
    ```
    smartctl -a /dev/sdx
    ```
    <Message type="note">

>       The identifier `sdx` refers to your hard disk, replacing x with a, b or c etc. depending on the disk you want to test.

Reviewer: ```suggestion
      The identifier `sdx` refers to your hard disk. Make sure that you replace `x` with `a`, `b` , `c`, etc. depending on the disk you want to test.
```
